### PR TITLE
chore: added robots.txt file ignoring previews

### PR DESCRIPTION
**Pull request by TomerFi**
Signed-off-by: Tomer Figenblat <tfigenbl@redhat.com>

### CODE REVIEW

File: .github/workflows/pr.yml [hunk 1]
```diff
@@ -59,6 +59,12 @@ jobs:
           mkdir -p pr-previews/pr-${{ github.event.number }}
           cp -rf _site/* pr-previews/pr-${{ github.event.number }}
 
+      - name: Patch robots.txt
+        run: |
+          tee pr-previews/pr-${{ github.event.number }}/robots.txt <<EOF
```
Comment (bennypowers) [inline .github/workflows/pr.yml]: no need to add to a subdir,, the root file is good enough

**Review by bennypowers — CHANGES_REQUESTED** [review]

File: .github/workflows/pr.yml (hunk 1 above)
Comment (TomerFi) [reply .github/workflows/pr.yml]: this is for the preview folder, the `pr-previews/pr-${{ github.event.number }}` folder should disallow everything.

the root will have the original robots.txt constructed from the `njk` template allowing everything besides the `pr-previews` folder.

Comment (bennypowers) [reply .github/workflows/pr.yml]: https://www.portent.com/blog/seo/robots-txt-mistakes-best-uses-guide.htm

yeah but there should be only one robots.txt per webroot

Comment (TomerFi) [reply .github/workflows/pr.yml]: cool - then in that case the one in the preview is redundant - I'll remove it.
:-)

Comment (TomerFi) [reply .github/workflows/pr.yml]: fixed.

**Review by bennypowers — APPROVED** [review]

**Pull request merged**